pkg/appmanager/resourceConfig.go

```diff
@@ -836,6 +838,234 @@ func setSourceAddrTranslation(snatPoolName string) SourceAddrTranslation {
 	}
 }
 
+func parseAppRootURLRewriteAnnotations(annotation string) map[string]string {
+	annotationValMap := make(map[string]string)
+
+	numSeps := strings.Count(annotation, ",")
+	numReps := strings.Count(annotation, "=")
+	if numSeps > 0 {
+		splits := strings.Split(annotation, ",")
+		for _, val := range splits {
+			if strings.Count(val, "=") != 1 {
+				log.Warningf("Annotation: %s value: %s not properly formatted should be replace-val=target-val, skipping", annotation, val)
+				continue
+			}
+			split := strings.Split(val, "=")
```

Comment:
what happens if i make this mistake? "/hello=/good-bye/hi=/bye"?

it looks like i'll configured this: map["/hello"] = "/good-bye/hi" which is not really correct. plus i won't know why i don't get the map["/hi"] = "/bye" redirect.

we should do more checking around these splits.

Reply:
This will fail since you have two equals signs thats the numSes == numReps check you do so even if you did "/hello=/good-bye/hi=/bye," it would still fail, but this could fail if you have ",/hello=/good-bye/hi=/bye,"  since num , == num =